adaptive_ai/services/prompt_classifier.py

```
@@ -77,111 +90,190 @@
-            "Science": [0.25, 0.40, 0.20, 0.10, 0.05, 0.00],
-            "Sensitive_Subjects": [0.30, 0.35, 0.15, 0.10, 0.05, 0.05],
-            "Shopping": [0.40, 0.30, 0.15, 0.10, 0.03, 0.02],
-            "Sports": [0.35, 0.30, 0.15, 0.10, 0.05, 0.05],
-            "Travel_and_Transportation": [0.30, 0.35, 0.15, 0.10, 0.05, 0.05],
-        }
+            return cast(List[float], scores)
 
-        if domain not in DOMAIN_WEIGHTS:
-            raise ValueError(f"Unknown domain: {domain}")
-        weights = DOMAIN_WEIGHTS[domain]
+    def process_logits(
```

Pylint (3.3.7)

[refactor] Too many local variables (30/15)

(R0914)